dev_ml_wsl.ps1

```diff
@@ -0,0 +1,35 @@
+# Description: Boxstarter Script
+# Author: Microsoft
+# Common dev settings for machine learning using Windows and Linux native tools
+
+Disable-UAC
+
+# Get the base URI path from the ScriptToCall value
+$bstrappackage = "-bootstrapPackage"
+$helperUri = $Boxstarter['ScriptToCall']
+$strpos = $helperUri.IndexOf($bstrappackage)
+$helperUri = $helperUri.Substring($strpos + $bstrappackage.Length).Trim(" '")
+$helperUri = $helperUri.Substring(0, $helperUri.IndexOf("'"))
+$helperUri = $helperUri.Substring(0, $helperUri.LastIndexOf("/"))
+$helperUri += "/scripts"
+write-host "helperUri is $helperUri"
+
```

Comment:
The above is ripping out the recipe script URI so we can dynamically pick up the helper scripts.  I tested it when running locally (C:/) and with a web link (http://)